packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -169,8 +169,22 @@ export class ListItem extends LitElement implements InteractiveComponent, Sortab
    */
   @property() moveToItems: MoveTo[] = [];
 
-  /** When `true`, the item is open to show child components. */
-  @property({ reflect: true }) open = false;
+  /**
+   * When `true`, the item is open to show child components.
```

Comment:
@geospatialem, would it be better to say: `When `true`, expands the component and its contents.`?